__mocks__/node-persist.js

```diff
@@ -0,0 +1,33 @@
+const storage = jest.genMockFromModule('node-persist');
+
+let data = {};
+
+storage.init = function() {
```

Comment:
I noticed that some places pass in options when they call init. If they're all just to specify a path, then it might be ok to ignore it.

I suppose another way to deal with this would be to have each test use a different storage file.

Reply:
Yeah, this is based on the fact that every use of init uses the same directory. If there were two calls to init with different directories whichever call occurred later would overwrite the first directory for that particular instance of Node.

An alternative approach to this PR would be to have every test use `test-settings/{randomly-generated-id}` as its directory instead of `test-settings`